plugins/modules/system/vdo.py

```diff
@@ -771,19 +679,15 @@ def run_module():
 
         diffsizeparams = {}
         for key in sizeparams.keys():
-            if module.params[key] is not None:
-                if str(sizeparams[key]) != module.params[key]:
-                    diffsizeparams[key] = module.params[key]
+            if module.params[key] is not None and sizeparams[key] != module.params[key]:
```

Comment:
Why is the `str` no longer needed?

Reply:
I remember backtracking the value of this variable in th code and concluding it would always be a `str`, but double checking now, this will eventually come from the YAML parsing in line 551 (new code), so it might - in fact it is likely it won't - be a string.